mitmproxy/master.py

```diff
@@ -117,15 +119,28 @@ def shutdown(self):
         self.should_exit.set()
         self.addons.trigger("done")
 
+    def _change_reverse_host(self, f):
+        if self.options.mode.startswith("reverse:"):
```

Comment:
Let's add a doc string since you asked on Slack what this does:
"When we load flows in reverse proxy mode, we adjust the target host to the reverse proxy destination for all flows we load. This makes it very easy to replay saved flows against a different host."

Reply:
👍